src/AppInstallerCLICore/Command.cpp

```diff
@@ -26,11 +26,12 @@ namespace AppInstaller::CLI
 
     Command::Command(
         std::string_view name,
+        std::vector<std::string_view> aliases,

```

Comment:
Maybe this should be `const &`?
Edit: Or `&&`

Reply:
I'm not sure I quite follow what you mean

Reply:
I think the type should be `const std::vector<std::string_view>&` or `std::vector<std::string_view>&&` to avoid creating a copy of the vector when the constructor is called.